tests/js/panelsTests.js

```diff
@@ -155,6 +155,22 @@ https://github.com/gpii/universal/LICENSE.txt
                     args: ["{textSize}", "{that}.options.testOptions.decreasedStep"],
                     spec: {path: "value", priority: "last"},
                     changeEvent: "{textSize}.applier.modelChanged"
+                }, {
+                    func: "{textSize}.applier.change",
+                    args: ["value", "{textSize}.options.range.max"]
+                }, {
+                    listener: "gpii.tests.textSizeTester.verifyButtonStates",
+                    args: ["{textSize}", true, false],
+                    spec: {path: "value", priority: "last"},
+                    changeEvent: "{textSize}.applier.modelChanged"
+                }, {
+                    func: "{textSize}.applier.change",
+                    args: ["value", "{textSize}.options.range.min"]
+                }, {
+                    listener: "gpii.tests.textSizeTester.verifyButtonStates",
+                    args: ["{textSize}", false, true],
+                    spec: {path: "value", priority: "last"},
+                    changeEvent: "{textSize}.applier.modelChanged"
```

Comment:
Can we also test button enable/disable states when the value is in the range of min and max but not min or max itself.